src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball = new Ball();
+        Lottery lottery = new Lottery();
+        for (int i = 0; i < 3; i++) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball);
+        }
```

Comment:
Consider using a constant for the number of balls to generate instead of a magic number '3'. This will make the code more readable and maintainable.